lambda/controller/validate-source/index.js

```diff
@@ -29,6 +31,19 @@ exports.handler = function(event, context, callback){
       }
     }
 
+    sourcedb.listTagsOfResource({ ResourceArn: sourceTable.Table.TableArn }, function(err, data) {
+      if(err){
+        return callback(new Error("Tags could not be listed for source table"));
```

Comment:
It may be a good idea to log the TableArn?

Reply:
I switched it to the aws err message instead since it provided more context (as well as the tableArn) while I tested this